docs/evm/networks.md

```diff
@@ -39,4 +39,4 @@ EVM on Flow has the following public RPC nodes available:
 | RPC Endpoint    | https://mainnet.evm.nodes.onflow.org   |
 | Chain ID        | 747                                    |
 | Currency Symbol | FLOW                                   |
-| Block Explorer  | https://flowdiver.io                   |
+| Block Explorer  | https://flowdiver.io                   |
```

Comment:
Should this also be https://eth.flowscan.io

Reply:
Left this alone because @Aliserag made the edits for this in his PR